BRB/run.py

```diff
@@ -27,15 +28,21 @@
    help='specify a custom ini file.',
    show_default=True
 )
-def run_brb(configfile):
+@click.option(
```

Comment:
Ideally a --stats options should have an accompanying --fid or path to a specific flow cell.

Reply:
Yes, I will make it so after testing and having a working version. I didn't want to deal with the inter-relationship (--fcid required after --stats), but I will add chatgpt about that later :P for now we are stuck with my own (mis-)understanding of this loop...